views/doc/en/deploys.md

```diff
@@ -18,7 +18,7 @@ be displayed on the ASYD web interface on the "Deploys" sections.
 * A "def" file (i.e. `data/deploys/LAMP/def`) with the definition of what the deploy will do -
 packages to install, commands to execute, configurations to upload, conditions, etc.
 * Optionally, a "def.sudo" file (i.e. `data/deploys/LAMP/def.sudo`) in case we want to
-execute it instead of the standard "def" when using a non-root user.
+execute it as super user instead of the standard "def" when using a non-root user.
```

Comment:
not really correct, on the def.sudo you still need to specify "sudo" on the exec lines that requires it, so the "def.sudo" -literally- gets executed (itself) WHEN you are using a non-root user on that host AND the file exists
Maybe needs to rephrase.